djredcap/management/subcommands/convert.py

```diff
@@ -39,11 +39,13 @@ class Command(BaseCommand):
     args = 'filename'
 
     json_filename = ''
+    #add a command line option -m allowing the user to set a filename for models
+    #filename is RELATIVE to the current working directory
+    BaseCommand.option_list = BaseCommand.option_list + (make_option('-m', '--modelfile', dest='modelfilename', help='filename to which models are written'),)
 
     option_list = BaseCommand.option_list + (
         make_option('--JSON', action='store', dest='json',
-                    help='Print a JSON file when using convert, default no'),
-    )
+                    help='Print a JSON file when using convert, default no'), make_option('-m', '--modelfile', dest='modelfilename', help='filename to which models are written'),)

```

Comment:
Also change the long option and `dest` to `--model-file` and `model_filename`, respectively, to increase the readability. Capitalize `filename` in the help description.